apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -174,3 +175,23 @@ $container-width: 309px;
     width: math.div($container-width - (2 * 16) - 12, 2);
   }
 }
+
+.centerExpandedCard {
+  display: flex;
+}
+
+.cardsContainer:nth-child(3n + 1) .centerExpandedCard {
+  justify-content: flex-start;
+}
+
+.cardsContainer:nth-child(3n + 2) .centerExpandedCard {
+  justify-content: center;
+}
+
+.cardsContainer:nth-child(3n + 3) .centerExpandedCard {
+  justify-content: flex-end;
+}
+
+.cardsContainer:nth-child(3n + 3) .arrowContainer:before {
+  background: $neutral_light;
+}
```

Comment:
Nit: does it still work if you combine them? (I can't remember if `.module.scss` allows this) like:
`.cardsContainer:nth-child(3n + 3) {`
`   .centerExpandedCard { justify....... }`
`   .arrowContainer:before { background...... }`
`}`